src/decaylanguage/decay/viewer.py

```diff
@@ -240,4 +240,7 @@ def _repr_svg_(self):
         """
         IPython display in SVG format.
         """
-        return self._graph._repr_svg_()
+        try:
+            return self._graph._repr_image_svg_xml()  # for graphviz >= 0.19
+        except AttributeError:
+            return self._graph._repr_svg_()  # for graphviz < 0.19
```

Comment:
This is exchanging a protocol for a private API. Instead we should use 
https://github.com/xflr6/graphviz/blob/723cf0213a112b458f3d5023f46daf0f2961ffa5/graphviz/jupyter_integration.py#L59

We should change over to providing `_repr_mimebundle_` instead, then build the mime bundle from `_repr_svg_()` if the `_graph` does not have the matching method.

Reply:
(And probably throw an error if the requested mime bundle is not svg and graphviz is old)

Reply:
Fair enough - this works but is not the most elegant.

You mean replace the `def _repr_svg_(self):` method with
```
    def _repr_mimebundle_(self, include=None, exclude=None, **kwargs):
        """
        IPython display in SVG format.
        """
        try:
            return self._graph._repr_mimebundle_(include=include, exclude=exclude, **kwargs)
        except AttributeError:
            return self._graph._repr_svg_()  # for graphviz < 0.19
```
?
This does not need to throw an exception since SVG will be picked up given the default `include=None` and will work with old graphviz versions, which I think is a must since 0.19 is very new.

Reply:
I think it's `return {"image/svg+xml": self._graph._repr_svg_()} `. I just meant it should throw an error if you request something other than svg, since this interface lets you request png and jpeg. But it's probably fine, it can still tell what got returned.

Reply:
Yep, was looking at that right now since the 2nd return is no longer a dict as expected :-). Let me test just in case ...

Reply:
OK